ba/tutorials/tour/abstract-types.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: tutorial
+title: Apstraktni tipovi
+
+disqus: true
+
+tutorial: scala-tour
+num: 22
+outof: 33
+language: ba
+---
+
+U Scali, klase su parameterizovane vrijednostima(parameteri konstruktora) i tipovima(ako su [generičke](generic-classes.html)).
```

Comment:
I will not comment on space before parenthesis any more. It should be fixed with search and replace.